rero/rero-ils-ui
# ui: add more `id=` attributes for patron and logo

* Adds `id=` attributes on patron profile tabs
* Adds `id=` attributes for logos (that permit to go to homepage)

Co-Authored-by: Olivier DOSSMANN <git@dossmann.net>

## Why are you opening this PR?

Cypress tests imply to target specific element from UI. This commit improves patron profile tabs and logo from UI with `id=` attributes.

## How to test?

* check that logo have an `id=homepage-logo` attribute (when you're logged in/out)
* check that patron profile have tabs that have `id=` attributes as `checkin-checkout`, `pick-up`, `fees`, `history`, etc.

## Code review check list
